pkg/platform/kube/test/suite.go

```diff
@@ -327,6 +330,16 @@ func (suite *KubeTestSuite) WaitForAPIGatewayState(getAPIGatewayOptions *platfor
 	suite.Require().NoError(err, "Api gateway did not reach its desired state")
 }
 
+func (suite *KubeTestSuite) GetDefaultIngressHost() string {
```

Comment:
code duplication with `pkg/nuctl/test/apigagteway_test.go:218`
Also, look on the UI how I configure `NUCTL_TEST_DEFAULT_APIGATEWAY_HOST`
You must do the same thing for the reason of how minikube proxy requests to functinos on CI builders